webdriver/tests/bidi/browsing_context/locate_nodes/locator.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+
+@pytest.mark.parametrize("type,value", [
+    ("css", "div"),
+    ("xpath", "//div"),
+    ("innerText", "foobarBARbaz")
+])
+@pytest.mark.asyncio
+async def test_find_by_locator(bidi_session, inline, top_context, type, value):
+    url = inline("""<div data-class="one">foobarBARbaz</div><div data-class="two">foobarBARbaz</div>""")
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"], url=url, wait="complete"
+    )
+
+    result = await bidi_session.browsing_context.locate_nodes(
+        context=top_context["context"],
+        locator={ "type": type, "value": value }
+    )
+
+    assert result["context"] == top_context["context"]
+    assert result["nodes"].length == 2
+    result_nodes = result["nodes"]
```

Comment:
It would be great if we could use the same kind of parametrization as in `evaluate/result_node.py`, which defines the expected nodes upfront and then just calls `recursive_compare()`.

Reply:
Should be done now. If I've effed it up somehow, we can adjust.